tests/features/media/browser_embed_code.feature

```diff
@@ -1,7 +1,7 @@
 @lightning @media @api @javascript @errors
 Feature: Creating media assets from within the media browser using embed codes
 
-  @video @twitter @instagram @test_module @2c43f38c
+  @video @twitter @instagram @with-module:lightning_test @2c43f38c
```

Comment:
New syntax! Now we can create smaller test modules for specific scenarios that need such scaffolding, rather than a gigantic monolith.